.travis.yml

```diff
@@ -0,0 +1,7 @@
+services:
+- docker
```

Comment:
This all looks reasonable, but I wasn't able to find a travis build for this repository and I don't see any passed validation checks on the PR - has this been set up in Travis?